tests/rt_gnu.conf

```diff
@@ -36,34 +35,34 @@ RUN     | fv3_gfs_v16_ras
-RUN     | fv3_gfs_v15p2_debug                                                                                                     |                | fv3         |
-RUN     | fv3_gfs_v16_debug                                                                                                       |                | fv3         |
-RUN     | fv3_gfs_v16_RRTMGP_debug                                                                                                |                | fv3         |
+RUN     | control_thompson_debug                                                                                                  |                | fv3         |
+RUN     | control_thompson_no_aero_debug                                                                                          |                | fv3         |
+RUN     | control_rrtmgp_debug                                                                                                    |                | fv3         |
+RUN     | control_ugwpv1_debug                                                                                                    |                | fv3         |
+RUN     | control_ras_debug                                                                                                       |                | fv3         |
 
-COMPILE | APP=ATM SUITES=HAFS_v0_hwrf_thompson,HAFS_v0_hwrf,FV3_GFS_v16_ugwpv1,FV3_GFS_v16_ras DEBUG=Y                            |                | fv3         |
+COMPILE | APP=ATM SUITES=HAFS_v0_hwrf_thompson,HAFS_v0_hwrf DEBUG=Y                                                               |                | fv3         |
 RUN     | fv3_HAFS_v0_hwrf_thompson_debug                                                                                         |                | fv3         |
 #RUN     | fv3_HAFS_v0_hwrf_debug                                                                                                  |                | fv3         |
 RUN     | fv3_esg_HAFS_v0_hwrf_thompson_debug                                                                                     |                | fv3         |
-RUN     | fv3_gfsv16_ugwpv1_debug                                                                                                 |                | fv3         |
-RUN     | fv3_gfs_v16_ras_debug                                                                                                   |                | fv3         |
 
-COMPILE | APP=ATM SUITES=FV3_GFS_2017_fv3wam 32BIT=Y MULTI_GASES=Y DEBUG=Y                                                        |                | fv3         |
-RUN     | fv3_multigases_debug                                                                                                    |                | fv3         |
+#COMPILE | APP=ATM SUITES=FV3_GFS_2017_fv3wam 32BIT=Y MULTI_GASES=Y DEBUG=Y                                                        |                | fv3         |
```

Comment:
Because the multigases test requires FV3_GFS_2017_fv3wam suite file, which was not used in the new RT test suite any more. The fv3_multigases test was commented out. Henry/Kate is preparing a PR to update multigases test using new ICs from GFSv16 analysis file, they can update the fv3_multigases_debug test in their PR. This way we can get rid of the old GFSv14 ICs.